### PR TITLE
Docs and output tinkering for commit check

### PR DESCRIPTION
**Pull request by chadwhitacre**
Quick follow-up to #1186, primarily to add the grouping we have in the other subscripts, and to give feedback when skipped.

### CODE REVIEW

File: install/parse-cli.sh [hunk 1]
```diff
@@ -4,14 +4,20 @@ show_help() {
   cat <<EOF
 Usage: $0 [options]
 
-Install Sentry with `docker compose`.
+Install Sentry with \`docker compose\`.
```
Comment (chadwhitacre) [inline install/parse-cli.sh:7]: Oops. Backticks spawn a subshell here, need to escape them. Regression in #1179.

File: install/parse-cli.sh [hunk 2]
```diff
@@ -4,14 +4,20 @@ show_help() {
...
+ --minimize-downtime    EXPERIMENTAL: try to keep accepting events for as long
+                          as possible while upgrading. This will disable cleanup
+                          on error, and might leave your installation in a
+                          partially upgraded state. This option might not reload
+                          all configuration, and is only meant for in-place
+                          upgrades.
+ --skip-commit-check    Skip the check for the latest commit when on the master
+                          branch of a \`self-hosted\` Git working copy.
+ --skip-user-prompt     Skip the initial user creation prompt (ideal for non-
+                          interactive installs).
```
Comment (chadwhitacre) [inline install/parse-cli.sh:20]: Opportunistic reformat for 80-char lines, sort alphabetically.

File: install/parse-cli.sh [hunk 3]
```diff
@@ -22,7 +28,8 @@ SKIP_COMMIT_CHECK="${SKIP_COMMIT_CHECK:-}"
 while (( $# )); do
   case "$1" in
     -h | --help) show_help; exit;;
-    --no-user-prompt) SKIP_USER_PROMPT=1;;
+    --no-user-prompt) SKIP_USER_PROMPT=1;;  # deprecated
+    --skip-user-prompt) SKIP_USER_PROMPT=1;;
```
Comment (chadwhitacre) [inline install/parse-cli.sh:32]: We shouldn't have `--no-*` for one and `--skip-*` for the other. Let's standardize on `--skip`.

**Review by BYK — APPROVED** [review]

**Review by aminvakil — APPROVED** [review]
Thanks for improving it!

**Pull request merged**